### PR TITLE
Update ECS to 8.5.2

### PR DESCRIPTION
**Pull request by kevinlog**
## Change Summary

Update the ECS version that Endpoint pulls in to `8.5.2`.  This is primarily done to get the updated field type for `env_vars`.

### Sample values

```
    "env_vars": [
      "NICK=test",
      "OTHER=why"
    ],
```

Sample document:

```json
{
  "@timestamp": "2022-11-29T20:16:45.9250502Z",
  "agent": {
    "id": "13146eac-6aa0-4332-86d7-21575ed20003",
    "type": "endpoint",
    "version": "8.6.0"
  },
  "data_stream": {
    "dataset": "endpoint.events.process",
    "namespace": "default",
    "type": "logs"
  },
  "ecs": {
    "version": "1.11.0"
  },
  "elastic": {
    "agent": {
      "id": "13146eac-6aa0-4332-86d7-21575ed20003"
    }
  },
  "event": {
    "action": "end",
    "category": [
      "process"
    ],
    "created": "2022-11-29T20:16:45.9250502Z",
    "dataset": "endpoint.events.process",
    "id": "MsIOUUNct/5WrYET+++++lrj",
    "kind": "event",
    "module": "endpoint",
    "sequence": 182036,
    "type": [
      "end"
    ]
  },
  "group": {
    "Ext": {
      "real": {
        "id": 1000,
        "name": "ubuntu"
      }
    },
    "id": 1000,
    "name": "ubuntu"
  },
  "host": {
    "architecture": "x86_64",
    "hostname": "ip-172-31-43-131",
    "id": "ec26a1ec5202d364ef6a05e10e414002",
    "ip": [
      "127.0.0.1",
      "::1",
      "172.31.43.131",
      "fe80::8b6:e0ff:fe0b:3b80"
    ],
    "mac": [
      "0a:b6:e0:0b:3b:80"
    ],
    "name": "ip-172-31-43-131",
    "os": {
      "Ext": {
        "variant": "Ubuntu"
      },
      "family": "ubuntu",
      "full": "Ubuntu 20.04.5",
      "kernel": "5.15.0-1022-aws #26~20.04.1-Ubuntu SMP Sat Oct 15 03:22:07 UTC 2022",
      "name": "Linux",
      "platform": "ubuntu",
      "type": "linux",
      "version": "20.04.5"
    }
  },
  "message": "Endpoint process event",
  "process": {
    "Ext": {
      "ancestry": [
        "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4NzM5LTE2Njk3NTI5NTM=",
        "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4NzM4LTE2Njk3NTI5NTM=",
        "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4NjIyLTE2Njk3NTI5NTE=",
        "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTcyNy0xNjY5NjY0NDM2",
        "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTEtMTY2OTY2NDM4MA=="
      ]
    },
    "args": [
      "sleep",
      "10"
    ],
    "args_count": 2,
    "command_line": "sleep 10",
    "end": "2022-11-29T20:16:45.9250502Z",
    "entity_id": "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4ODkwLTE2Njk3NTI5OTU=",
    "entry_leader": {
      "args": [
        "-bash"
      ],
      "args_count": 1,
      "entity_id": "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4NzM5LTE2Njk3NTI5NTM=",
      "entry_meta": {
        "source": {
          "ip": "73.134.228.157"
        },
        "type": "sshd"
      },
      "executable": "/bin/bash",
      "group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "interactive": true,
      "name": "bash",
      "parent": {
        "entity_id": "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4NzM4LTE2Njk3NTI5NTM=",
        "pid": 18738,
        "start": "2022-11-29T20:15:53.47Z"
      },
      "pid": 18739,
      "real_group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "real_user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "same_as_process": false,
      "start": "2022-11-29T20:15:53.59Z",
      "supplemental_groups": [
        {
          "id": 4,
          "name": "adm"
        },
        {
          "id": 20,
          "name": "dialout"
        },
        {
          "id": 24,
          "name": "cdrom"
        },
        {
          "id": 25,
          "name": "floppy"
        },
        {
          "id": 27,
          "name": "sudo"
        },
        {
          "id": 29,
          "name": "audio"
        },
        {
          "id": 30,
          "name": "dip"
        },
        {
          "id": 44,
          "name": "video"
        },
        {
          "id": 46,
          "name": "plugdev"
        },
        {
          "id": 117,
          "name": "netdev"
        },
        {
          "id": 118,
          "name": "lxd"
        }
      ],
      "tty": {
        "char_device": {
          "major": 136,
          "minor": 0
        }
      },
      "user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "working_directory": "/home/ubuntu"
    },
    "env_vars": [
      "NICK=test",
      "OTHER=why"
    ],
    "executable": "/usr/bin/sleep",
    "exit_code": 0,
    "group": {
      "id": 1000,
      "name": "ubuntu"
    },
    "group_leader": {
      "args": [
        "sleep",
        "10"
      ],
      "args_count": 2,
      "entity_id": "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4ODkwLTE2Njk3NTI5OTU=",
      "executable": "/usr/bin/sleep",
      "group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "interactive": true,
      "name": "sleep",
      "pid": 18890,
      "real_group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "real_user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "same_as_process": true,
      "start": "2022-11-29T20:16:35.9Z",
      "supplemental_groups": [
        {
          "id": 4,
          "name": "adm"
        },
        {
          "id": 20,
          "name": "dialout"
        },
        {
          "id": 24,
          "name": "cdrom"
        },
        {
          "id": 25,
          "name": "floppy"
        },
        {
          "id": 27,
          "name": "sudo"
        },
        {
          "id": 29,
          "name": "audio"
        },
        {
          "id": 30,
          "name": "dip"
        },
        {
          "id": 44,
          "name": "video"
        },
        {
          "id": 46,
          "name": "plugdev"
        },
        {
          "id": 117,
          "name": "netdev"
        },
        {
          "id": 118,
          "name": "lxd"
        }
      ],
      "tty": {
        "char_device": {
          "major": 136,
          "minor": 0
        }
      },
      "user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "working_directory": "/home/ubuntu"
    },
    "hash": {
      "md5": "fcba58db24e5e3672c4d70a3bb01d7a4",
      "sha1": "cae542290d1bb5c91c637350e0a633f71fd2a6e4",
      "sha256": "45cf3208dc6704e806bbc5d776e884b5487744bd75171a93930c94e9b9b20ebb"
    },
    "interactive": true,
    "name": "sleep",
    "parent": {
      "args": [
        "-bash"
      ],
      "args_count": 1,
      "command_line": "-bash",
      "entity_id": "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4NzM5LTE2Njk3NTI5NTM=",
      "executable": "/bin/bash",
      "group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "interactive": true,
      "name": "bash",
      "pid": 18739,
      "real_group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "real_user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "start": "2022-11-29T20:15:53.59Z",
      "supplemental_groups": [
        {
          "id": 4,
          "name": "adm"
        },
        {
          "id": 20,
          "name": "dialout"
        },
        {
          "id": 24,
          "name": "cdrom"
        },
        {
          "id": 25,
          "name": "floppy"
        },
        {
          "id": 27,
          "name": "sudo"
        },
        {
          "id": 29,
          "name": "audio"
        },
        {
          "id": 30,
          "name": "dip"
        },
        {
          "id": 44,
          "name": "video"
        },
        {
          "id": 46,
          "name": "plugdev"
        },
        {
          "id": 117,
          "name": "netdev"
        },
        {
          "id": 118,
          "name": "lxd"
        }
      ],
      "tty": {
        "char_device": {
          "major": 136,
          "minor": 0
        }
      },
      "user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "working_directory": "/home/ubuntu"
    },
    "pid": 18890,
    "previous": [
      {
        "args": [
          "-bash"
        ],
        "args_count": 1,
        "executable": "/bin/bash"
      }
    ],
    "real_group": {
      "id": 1000,
      "name": "ubuntu"
    },
    "real_user": {
      "id": 1000,
      "name": "ubuntu"
    },
    "session_leader": {
      "args": [
        "-bash"
      ],
      "args_count": 1,
      "entity_id": "MTMxNDZlYWMtNmFhMC00MzMyLTg2ZDctMjE1NzVlZDIwMDAzLTE4NzM5LTE2Njk3NTI5NTM=",
      "executable": "/bin/bash",
      "group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "interactive": true,
      "name": "bash",
      "pid": 18739,
      "real_group": {
        "id": 1000,
        "name": "ubuntu"
      },
      "real_user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "same_as_process": false,
      "start": "2022-11-29T20:15:53.59Z",
      "supplemental_groups": [
        {
          "id": 4,
          "name": "adm"
        },
        {
          "id": 20,
          "name": "dialout"
        },
        {
          "id": 24,
          "name": "cdrom"
        },
        {
          "id": 25,
          "name": "floppy"
        },
        {
          "id": 27,
          "name": "sudo"
        },
        {
          "id": 29,
          "name": "audio"
        },
        {
          "id": 30,
          "name": "dip"
        },
        {
          "id": 44,
          "name": "video"
        },
        {
          "id": 46,
          "name": "plugdev"
        },
        {
          "id": 117,
          "name": "netdev"
        },
        {
          "id": 118,
          "name": "lxd"
        }
      ],
      "tty": {
        "char_device": {
          "major": 136,
          "minor": 0
        }
      },
      "user": {
        "id": 1000,
        "name": "ubuntu"
      },
      "working_directory": "/home/ubuntu"
    },
    "start": "2022-11-29T20:16:35.9Z",
    "supplemental_groups": [
      {
        "id": 4,
        "name": "adm"
      },
      {
        "id": 20,
        "name": "dialout"
      },
      {
        "id": 24,
        "name": "cdrom"
      },
      {
        "id": 25,
        "name": "floppy"
      },
      {
        "id": 27,
        "name": "sudo"
      },
      {
        "id": 29,
        "name": "audio"
      },
      {
        "id": 30,
        "name": "dip"
      },
      {
        "id": 44,
        "name": "video"
      },
      {
        "id": 46,
        "name": "plugdev"
      },
      {
        "id": 117,
        "name": "netdev"
      },
      {
        "id": 118,
        "name": "lxd"
      }
    ],
    "tty": {
      "char_device": {
        "major": 136,
        "minor": 0
      }
    },
    "user": {
      "id": 1000,
      "name": "ubuntu"
    },
    "working_directory": "/home/ubuntu"
  },
  "user": {
    "Ext": {
      "real": {
        "id": 1000,
        "name": "ubuntu"
      }
    },
    "id": 1000,
    "name": "ubuntu"
  }
}
```

## Release Target

8.6.0

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes
- [ ] If these field(s) are "exception"-able, I made a companion PR to Kibana adding it (see [Readme](https://github.com/elastic/endpoint-package#exceptionable))
- [ ] If this is a `metadata` change, I also updated both transform destination schemas to match

### CODE REVIEW

**Review by lrishi — APPROVED** [review]
lgtm 👍🏽

File: package/endpoint/data_stream/alerts/fields/fields.yml [hunk 1]
```diff
@@ -4226,9 +4226,7 @@
       ignore_above: 1024
       description: 'Use the `os.type` field to categorize the operating system into one of the broad commercial families.
 
-        One of these following values should be used (lowercase): linux, macos, unix, windows.
-
-        If the OS you''re dealing with is not in the list, the field should not be populated. Please let us know by opening an issue with ECS, to propose its addition.'
+        If the OS you''re dealing with is not listed as an expected value, the field should not be populated. Please let us know by opening an issue with ECS, to propose its addition.'
```
[the remark this answers is not in the document]
Comment (nicholasberlin) [reply package/endpoint/data_stream/alerts/fields/fields.yml:4229]: well, I see it a lot, so i'm guessing this is fine.

**Review by nicholasberlin — APPROVED** [review]

File: package/endpoint/data_stream/alerts/fields/fields.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinlog) [reply package/endpoint/data_stream/alerts/fields/fields.yml:4229]: this is pulled directly in from ECS, so I'm leaving as is.

**Review by paul-tavares — APPROVED** [review]

**Pull request merged**